applications/FSIApplication/custom_utilities/mvqn_convergence_accelerator.hpp

```diff
@@ -217,6 +218,8 @@ class MVQNFullJacobianConvergenceAccelerator: public ConvergenceAccelerator<TSpa
     ///@name Friends
     ///@{
 
+    friend class IBQNMVQNConvergenceAccelerator<TSparseSpace, TDenseSpace>;
```

Comment:
🙈 

can you add a comment as to why this is necessary?

Is this why you added the [protected interfaces in the BaseClass](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/solving_strategies/convergence_accelerators/convergence_accelerator.h#L198-L211)?

Reply:
Yeah. I was expecting this coment :smile: 

Yes. My original idea was to intentionally difficult the access to the interface Jacobians to avoid these to be misused. Since the `IBQNMVQNConvergenceAccelerator` derives from the `ConvergenceAccelerator` base class, this requires to make it a `friend` of the standard `MVQN`.

In here I see two things:
1. We could derive the `IBQNMVQN` from the `MVQN`. However this would preclude the creation of an eventual intermediate `InterfaceBlockQuasiNewtonConvergenceAccelerator` class unifying the API of all the interface block Quasi-Newton convergence accelerators.
2. Maybe it is not worth it to do this and we can just make the access to the Jacobians `public`.